### PR TITLE
Add YCbCr full range support

### PR DESCRIPTION
**Pull request by fabiosky**
Added YCbCr full range (0-255) support.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by nacho** [comment]
LGTM

File: c_tests/unit.c [hunk 1]
```diff
@@ -17,6 +17,7 @@
  */
 
 #include <stdlib.h>
+#include <stdbool.h>
```
Comment (pbor) [inline c_tests/unit.c:20]: does this build on all platforms we support?

**Review by pbor — APPROVED** [review]
Clearly I have no insight on the actual values of the coefficient and on the asm implementation, but this all look reasonable.

I left a minor question below.

If possible I would squash the cargo fmt fixup and split the PR in two commits: the full range implementation and the 0.4 version bump. But if that is a hassle, squashing everything in a single patch that is fine with me as well.

File: c_tests/unit.c (hunk 1 above)
Comment (fabiosky) [reply c_tests/unit.c:20]: Address in https://github.com/aws/dcv-color-primitives/pull/58

**Pull request closed without merging**